source/faq.txt

```diff
@@ -41,6 +41,59 @@ For applications that require a mix of the new and legacy APIs, ``com.mongodb.Mo
 - Configuration with ``MongoClientSettings`` and ``ConnectionString``, the only difference being that you create instances via constructors instead of a factory class.
 - CRUD API using ``MongoDatabase``, and from there, ``MongoCollection``.  You can access this API via the ``getDatabase()`` method.
 
+.. _faq-legacy-connection:
+
+How do I connect to my MongoDB instance with the legacy API?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The following example shows how to connect to a MongoDB instance with the
+legacy API and the current API.
+
+Imagine we are connecting to a collection that contains only this document:
+
+.. code-block:: json
+   
+   {"_id": 1, "val": 1}
+
+.. tabs::
```

Comment:
Suggestion: Since the question is around legacy API, I don't think you should show the current API (we use it in all our pages anyways)

Reply:
I am not sure I agree with this suggestion. Here is my reasoning:

I think it is helpful to toggle back and forth between the API versions and see line-by-line how you need to change your code between the legacy and current api. I think it makes the example easier to follow as there becomes a natural 1-1 mapping between legacy API and current API code (allowing readers to more naturally apply current API docs to legacy API concepts).